src/librustc/traits/object_safety.rs

```diff
@@ -462,7 +462,7 @@ impl<'tcx> TyCtxt<'tcx> {
             .filter(|(_, item)| item.kind == ty::AssocKind::Type)
             .collect::<Vec<_>>();
 
-        // existential predicates need to be in a specific order
+        // Existential predicates need to be in a specific order.
```

Comment:
Same here, please do not commit formatting changes, especially in files that are otherwise unchanged.

Reply:
“Pointless” is your opinion, but you’re right, they are not too close to functional changes so I’ll remove them.

Reply:
Done